rlberry/benchmarks/benchmark_utils.py

```diff
@@ -0,0 +1,151 @@
+import requests
+from tempfile import mkdtemp
+import os
+import shutil
+
+
+# TODO : convert external benchmark to DataFrame that match the input of rlberry.manager.comparaison.py -> compare_agents_data()
+# TODO : Download the external benchmark to a specific folder (or new rlberrygithub?), except if they are stable (huggingface/github)
+
+benchmark_list = {
+    "Google Atari bucket": "https://console.cloud.google.com/storage/brow",
+    "SB3 zoo": "https://github.com/DLR-RM/rl-baselines3-zoo/tree/master/logs/benchmark",
+    "cleanrl": "https://wandb.ai/openrlbenchmark/openrlbenchmark/reportlist",
+}
+
+
+def import_from_google_atari_bucket():
+    """import benchmark from Google Atari bucket
+
+    Parameters
+    -----------
+    x_vec : numpy.ndarray
+        numpy 1d array to be searched in the bins
+    bins : list
+        list of numpy 1d array, bins[d] = bins of the d-th dimension
+
+
+    Returns
+    --------
+    index (int) corresponding to the position of x in the partition
+    defined by the bins.
+    """
+    print("TODO")
+
+
+def import_from_cleanrl():
+    print("TODO")
+
+
+def import_from_hugingface():
+    print("TODO")
```

Comment:
Maybe comment this out for now, so that it does not change the coverage. And anyway if it is not implemented we don't want people to see it in the completion menu of ide.

Reply:
yes